lab-daniel/src/lib/reader.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const fs = require('fs');
+
+const fileReader = module.exports = {};
+
+fileReader.readAsync = (paths, callback) => {
+  try {
+    fs.readFile(
+      paths,
+      (error, result) => {
+        if (typeof paths !== 'string') {
+          throw error;
+        }
+        return callback(result.toString('utf8'));
+      },
```

Comment:
**TODO**: You need to get rid of this comma at Line 16 because your code does not execute because this is not valid JS. I think you may have added that comma due to linter yelling at you too. I think Linter got confused with your indenting and thought the curly bracket at Line 16 was an object as opposed to closing curly braces for your callback function, so it gave you that "trailing comma" warning.

Please rewrite this code without all that extra indentation and sans comma, and your code works.